content/2024-04-15-t8code_version200_released.md

```diff
@@ -0,0 +1,28 @@
+Title: t8code-2.0.0 released 
+Date: 2024-04-15 14:00 
+Category: News 
+Authors: Jakob Fußbroich
+
+We're thrilled to unveil our second major release: t8code v2.0.0! This latest version is packed with enhancements, polished interfaces, and exciting new features that will enhence your t8code experience.
+
+<img src="images/release200/Release_2_0.png" height="600" />
+
+Among the highlights are:
+
+- Introducing batch-based searching, revolutionizing the way you find what you need.
+
+- Streamlined unit testing with the seamless integration of GoogleTest, making testing a breeze.
+
+- Embark on a journey through new Lagrangian geometry, opening up boundless possibilities.
+
+- Dive into the realm of curved tetrahedra, adding depth and sophistication to your designs.
+
+- Enjoy the freedom of checking an arbitrary number of points for point_inside verification.
+
+- Harness the power of MPI-pack support for elements, facilitating efficient communication and collaboration.
+
+- Experience the elegance of a cleaned-up element interface, simplifying your workflow and enhancing productivity.
```

Comment:
Imho this is a bit too much and sounds too much like chatgpt. Especially the revolutionizing is too much.